CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
It used to be that every merge to master did a staging deploy. Are you sure that changed?